src/store/mutations.js

```diff
@@ -32,7 +32,17 @@ const mutations = {
     });
     Vue.delete(state.groups, id);
   },
-
+  togglePartialGroup(state, id) {
+    state.groups[id].partial = !state.groups[id].partial;
+  },
```

Comment:
This is a mistake, `partial` doesn't need to be into the state. It could be a virtual status. If some `parts` are set, it will be open by default, if not it will be closed by default